app/views/budgets/investments/_author_actions.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if show_author_actions?(investment) %>
+  <div class="sidebar-divider"></div>
+  <div class="show-actions-menu">
+    <h2><%= t("budgets.investments.show.author") %></h2>
+    <% if can?(:edit, investment) %>
+      <%= link_to t("shared.edit"),
+                  edit_budget_investment_path(investment.budget, investment),
+                  method: :get, class: "button hollow expanded" %>
+    <% else %>
+        <%= link_to image_path(investment.image),
+                    method: :delete,
+                    class: "button hollow alert expanded" do %>
+                      <span class="icon-image"></span>
+                      <%= t("images.remove_image") %>
+                    <% end %>
```

Comment:
IMHO the `<% end %>` is easier to read if it's aligned with the beginning ot the `<%= link_to line` :thinking:. What do you think?